### PR TITLE
Top community questions

### PR DESCRIPTION
**Pull request by GoodPassiveMan**

### CODE REVIEW

File: app/containers/AccountProvider/saga.js [hunk 1]
```diff
@@ -304,6 +312,31 @@ export function* updateAccWorker({ eos }) {
...
+      if (info) {
+        const isAdmin = isUserAdmin(info.properties);
+
+        if (isAdmin) {
+          yield put(getUserProfileSuccess({ ...profileInfo, isAdmin: true }));
+        }
+      }
+    }
+    // eslint-disable-next-line no-empty
+  } catch (e) {}
```
Comment (sergeyi23) [inline app/containers/AccountProvider/saga.js:337]: Is it expected to hide error here? Is it not critical?

File: app/translations/ru.json [hunk 2]
```diff
@@ -785,6 +785,8 @@
   "app.containers.Questions.thisIsNewCommunity":
     "Это новое сообщество, Вы можете стать первым и Ваш вопрос может стать самым популярным.",
   "app.containers.Questions.readMyFeed": "Читать мой фид",
+  "app.containers.Questions.topQuestionPopover":
+    "Частозадаваемый вопрос этого сообщества",
```
Comment (sergeyi23) [inline app/translations/ru.json:789]: space is missing "Часто задаваемый"

File: app/translations/en.json [hunk 3]
```diff
@@ -752,6 +752,8 @@
   "app.containers.Questions.thisIsNewCommunity":
     "This is a new community, you can be the first post a question. Your question may become the most popular.",
   "app.containers.Questions.readMyFeed": "Read my feed",
+  "app.containers.Questions.topQuestionPopover":
+    "A question that specified as top and frequently asked at this community.",
```
Comment (sergeyi23) [inline app/translations/en.json:756]: Maybe let's change the wording to be simple and similar to Russian version - "Top question of this community"

File: app/containers/AccountProvider/saga.js (hunk 1 above)
Comment (GoodPassiveMan) [reply app/containers/AccountProvider/saga.js:337]: Yeah, it's not critical

**Pull request merged**